tock/tock/urls.py

```diff
@@ -13,28 +11,40 @@
 
 urlpatterns = patterns(
     '',
-    url(r'^$', hours.views.ReportingPeriodListView.as_view(),
-        name='ListReportingPeriods'),
-    url(r'^reporting_period/', include("hours.urls.timesheets",
-        namespace="reportingperiod")),
-    url(r'^reports/', include("hours.urls.reports", namespace="reports")),
-    url(r'^employees/', include("employees.urls", namespace="employees")),
+    url(r'^$',
+        hours.views.ReportingPeriodListView.as_view(),
+        name='ListReportingPeriods'
+    ),
+    url(r'^reporting_period/', include(
+        'hours.urls.timesheets',
+        namespace='reportingperiod'
+    )),
+    url(r'^reports/', include(
+        'hours.urls.reports',
+        namespace='reports'
+    )),
+    url(r'^employees/', include(
+        'employees.urls',
+        namespace='employees'
+    )),
+    url(r'^projects/', include(projects.urls)),
 
     # TODO: version the API?
     url(r'^api/', include(api.urls)),
 
-    # project views
-    url(r'^projects/', include(projects.urls)),
-
-    # Uncomment the next line to enable the admin:
+    # Enable the Django admin.
     url(r'^admin/', include(admin.site.urls)),
 )
 
 
+# Enable Django Debug Toolbar only if in DEBUG mode.
 if settings.DEBUG:
     import debug_toolbar
```

Comment:
Ah, I see what you're saying, but in this case I think it's warranted for two reasons:

* The project requirements (the dev requirements in particularly) specify Django Debug Toolbar as a dependency and therefore I think it's reasonable to expect that a person would have it available as a part of just setting up the project for themselves.
* This is the recommended approach in the [Django Debug Toolbar documentation for the explicit configuration](https://django-debug-toolbar.readthedocs.io/en/1.4/installation.html#explicit-setup); the explicit configuration is already the recommended way to go, but in the upcoming 1.5 version that will be even more so.

That said, Django Debug Toolbar does/did do this automatically for you before with the simple configuration, but it's not an ideal setup and not meant for production.